src/modules/SwapModule.ts

```diff
@@ -107,7 +117,7 @@ export class SwapModule implements IModule {
     const coinToDecimals = +sortedToCoinInfo.data.decimals;
 
     let rate;
-    if (!params.amount) {
+    if (params.amount.comparedTo(0) == 0) {
```

Comment:
comparedTo() returns number type so it is no need to use double equal sign to escape type checking.